baselineprofile/build.gradle.kts

```
@@ -0,0 +1,67 @@
+import com.android.build.api.dsl.ManagedVirtualDevice
+
+plugins {
+    alias(libs.plugins.android.test)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.baselineprofile)
+}
+
+android {
+    namespace = "com.terning.point.baselineprofile"
+    compileSdk = libs.versions.compileSdk.get().toInt()
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+
+    kotlinOptions {
+        jvmTarget = "11"
```

```suggestion
        jvmTarget = libs.versions.jvmTarget.get()
```
이친구도 이렇게 버전 관리해주는 것은 어떨까요??

---

`libs.versions.jvmTarget.get(`)은 1.8인데 `compileOptions`은 11이라 버전이 맞지 않아서 "11"로 넣어줬었어요!
그래도 jvmTarget은 맞추는 게 좋을 것 같아서 이건 1.8로 바꿔둘게요~!